src/state/psarstake/utils.ts

```diff
@@ -319,3 +327,66 @@ export function useDefaultSarClaimOrCompound() {
     wrappedOnDismiss,
   };
 }
+
+export function createURI(
```

Comment:
please add some comment on 
- why we needed this function
- what is source of this function ( if needed point back to contract code )
- brief on what it is doing
- if it is only for hedera then rename it to indicate it

Reply:
`if it is only for hedera then rename it to indicate it`
No, it's for all chains, is fallback function, if not exist a URI, it will generate one to show the image to user and the similar attributes returned by smart contract